spot_description/urdf/spot_macro.xacro

```diff
@@ -409,7 +409,9 @@
         <xacro:include filename="$(find spot_description)/urdf/accessories.urdf.xacro" />
 
         <!-- Optional custom includes. -->
-        <xacro:include filename="$(optenv SPOT_URDF_EXTRAS empty.urdf)" />
+
+        <xacro:property name="default_empty_urdf" value="$(find spot_description)/urdf/empty.urdf" />
+        <xacro:include filename="$(optenv SPOT_URDF_EXTRAS ${default_empty_urdf})" />
```

Comment:
Before I added this, I was getting some weird errors that empty.urdf was not found when trying to use this macro in the ros2 control package. The URDF that gets generated by default from spot_description is still exactly the same after this change

Reply:
Your fix is correct, but what this is doing is pretty weird.  With the empty.urdf included, the generated urdf will be unchanged, but it someone has the SPOT_URDF_EXTRAS environment variable set to a different URDF, the generated URDF can be very different.  Seems like this would have been better to pass into the xacro as an arg.

Reply:
Agree with this, but as this is open source I am worried about changing the interface in case somebody is using this (although the chances are pretty low)